src/common/hooks/useLoginGuard/index.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { act, renderHook } from '@testing-library/react-hooks'
+import { shallow } from 'enzyme'
+
+describe('useLoginGuard', () => {
+  const isLoggedInSpy = jest.fn()
+
+  jest.doMock('@/store/userStore', () => ({
+    userStore: {
+      isLoggedIn: isLoggedInSpy,
+    },
+  }))
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('should return true if user is logged in', async () => {
+    isLoggedInSpy.mockReturnValue(true)
+
+    const { useLoginGuard } = await import('.')
```

Comment:
Any reason to import it here instead of at the header?

Reply:
You have to import thing AFTER mocking the functions or modules. The only case you can import at the header is you don't have to mock any functions or modules